github_actions/run.sh

```diff
@@ -2,16 +2,19 @@
 
 install_fossa()
 {
-  echo "Installing FOSSA CLI $FOSSA_VERSION"
   FOSSA_BIN_DIR="${FOSSA_BIN_DIR:=/usr/local/bin}"
   url="https://raw.githubusercontent.com/fossas/fossa-cli/master/install-latest.sh"
 
-  if [ $FOSSA_VERSION == "v1" ]; then
-    url="https://raw.githubusercontent.com/fossas/fossa-cli/master/install.sh"
+  ret=1
+  if [ -z $FOSSA_CLI_VERSION ]; then
+    echo "Installing FOSSA CLI v$FOSSA_VERSION"
+    curl -H 'Cache-Control: no-cache' $url | bash -s -- -b $FOSSA_BIN_DIR v$FOSSA_CLI_VERSION
+  else
+    echo "Installing latest FOSSA CLI version"
+    curl -H 'Cache-Control: no-cache' $url | bash -s -- -b $FOSSA_BIN_DIR
   fi
```

Comment:
@Jman420 This PR broke Rave PR checker
I don't think this is correct logic
What is the desired behavior here?

Reply:
To provide a mechanism for specifying the version of the FOSSA CLI that the pipeline wants to use : https://github.com/fossas/fossa-cli/blob/master/docs/walkthroughs/installing-fossa-cli.md#linux-or-macos